protocols/v2/noise-sv2/src/handshake.rs

```diff
@@ -30,6 +30,8 @@
 // mandatory for communication across external networks (e.g., between a local mining proxy and a
 // remote pool).
 
+use alloc::{string::String, vec::Vec};
+
```

Comment:
As we are now having support for both env, we should consider to have environment specific imports.

Reply:
std::string::String is falling into alloc::string::String anyway

same for Vec